test/e2e/framework/statefulset/fixtures.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strconv"
 
 	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
So stuff like this is orthogonal to your PR, in the future please stick to 1 change at a time.

Reply:
> There is a ton of cleanup work that can/should be done in those utility functions if you wanted to open an issue to track.

Yes, the service util has a thousand lines of code which is hard to refactor them all in one PR.

> So stuff like this is orthogonal to your PR, in the future please stick to 1 change at a time.

Sure, I'll follow your suggestion next time.

Reply:
Agree with Tim here, I'm mostly in-different about the change itself but makes reviewing harder

Reply:
actually for this case, there's already an import alias to `corev1` which I think is better?

Reply:
> actually for this case, there's already an import alias to `corev1` which I think is better?

We have an import alias file now which defines how to name the imported package. `./hack/update-import-alias.sh` could help us to follow the import alias convention.

FYI: https://github.com/kubernetes/kubernetes/pull/78836